templates/terraform/post_create/sshkeyfingerprint.erb

```diff
@@ -0,0 +1,6 @@
+// `fingerprint` is autogenerated from the api so needs to be set post-create
```

Comment:
Got it, this isn't working as expected because we're handling an [`SSHPublicKey`](https://cloud.google.com/compute/docs/oslogin/rest/Shared.Types/SshPublicKey) but the API is returning a [`LoginProfile`](https://cloud.google.com/compute/docs/oslogin/rest/v1/LoginProfile) (wrapped under the `loginProfile` key: https://cloud.google.com/compute/docs/oslogin/rest/v1/users/importSshPublicKey#response-body)

The `res` object is a `map[string]interface{}` representation of the JSON response, and we can traverse it to retrieve the fingerprint value. https://github.com/GoogleCloudPlatform/magic-modules/blob/master/templates/terraform/post_create/storage_hmac_key.go.erb is an example of a case where we step into a single object.

Here, we'll want to step into `loginProfile`, and then `sshPublicKeys`. I believe `sshPublicKeys` will be a `map[string]interface{}` pointing to a `map[string]interface{}`. We'll then want to find the entry in `sshPublicKeys`, whose `key` value matches `d.Get("key")`, and record that fingerprint.